Logger/BaseLogger.cs

```diff
@@ -1,7 +1,15 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public abstract class BaseLogger
     {
+        public string className 
```

Comment:
Properties should be PascalCase
```suggestion
        public string ClassName 
```